src/tasks/AutoCombatLogic.py

```
@@ -32,6 +71,12 @@ def run(self, start_sleep: float = None, no_battle: bool = False):
                 task._last_no_combat_log_time = now
             task.sleep(0.5)
             return False
+
+        # 初始化普通战斗配置属性（排轴与普通模式共用）
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)